src/providers/BackgroundReplacementProvider/index.tsx

```diff
@@ -179,10 +182,29 @@ export const BackgroundReplacementProvider: FC<
     }
   };
 
+  const changeBackgroundReplacementImage = async (
```

Comment:
So this only allows to change background image and they could not change back to background color right?

Reply:
It allows to change back to color, I set the name to `changeBackgroundReplacementImage` as the method in the js library is `setImageBlob`, color is also a kind of image.

Reply:
Kinda confusing but we could clarify that in the storybook

Reply:
Storybook updated.